packages/toolkit/src/query/core/buildMiddleware/polling.ts

```diff
@@ -72,16 +73,16 @@ export const buildPollingHandler: InternalHandlerBuilder = ({
 
     const nextPollTimestamp = Date.now() + lowestPollingInterval
 
-    const currentInterval: typeof currentPolls[number] = (currentPolls[
-      queryCacheKey
-    ] = {
+    currentPolls[queryCacheKey] = {
       nextPollTimestamp,
       pollingInterval: lowestPollingInterval,
       timeout: setTimeout(() => {
-        currentInterval!.timeout = undefined
-        api.dispatch(refetchQuery(querySubState, queryCacheKey))
+        if (document.hasFocus() || !skipPollOnFocusLost) {
```

Comment:
this is a web-specific API, so won't work for React Native users (for one example)

instead, you'd need to make use of the state that `setupListeners` modifies:
```suggestion
        if (api.getState()[reducerPath].config.focused || !skipPollOnFocusLost) {
```

Reply:
Thanks, was definitely curious why the refetch behaviours opted into that.
Updated PR to reflect this change with:
- Tests changed
- Documentation in apiState.ts changed
- Behaviour changed